cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -1,13 +1,13 @@
 define(
     [
         'jquery', 'underscore',
+        'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
         'js/views/video/transcripts/utils',
         'js/views/video/transcripts/metadata_videolist', 'js/models/metadata',
         'js/views/abstract_editor',
-        'common/js/spec_helpers/ajax_helpers',
         'xmodule'
     ],
-function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, AjaxHelpers) {
+function ($, _, AjaxHelpers, Utils, VideoList, MetadataModel, AbstractEditor) {
```

Comment:
Nice. Yeah, I'm in favor of something similar in the JS dev guide. JS has no stdlib so it would just be:

1. third party node_modules/vendor requires
2. first party node_modules (e.g., UI Toolkit
3. local requires